packages/components/src/InputNumber/InputNumber.test.tsx

```diff
@@ -63,7 +63,7 @@ test("it should call the validation with undefined as a success", () => {
     />,
   );
 
-  expect(validationHandler).toHaveBeenCalledWith(undefined);
+  expect(validationHandler).toHaveBeenCalledWith("");
```

Comment:
Test name shouldn't be `it should call the validation with undefined as a success`